api/src/Blocks.ts

```diff
@@ -43,6 +66,20 @@ export class GearBlock {
     return block.block.header.number;
   }
 
+  /**
+   * Get timestamp of block
+   * @param hashOrNumber hash or number of particular block
+   * @returns
+   */
+  async getBlockTimestamp(hashOrNumber: `0x${string}` | Uint8Array | number): Promise<Compact<u64>> {
+    const block = await this.get(hashOrNumber);
+    const tsAsU8a = block.block.extrinsics.filter(
+      (value) => value.method.method === 'set' && value.method.section === 'timestamp',
+    )[0].data;
```

Comment:
```suggestion
    const tsAsU8a = block.block.extrinsics.find(
      (value) => value.method.method === 'set' && value.method.section === 'timestamp',
    ).data;
```
Is there a need to use `filter`? `find` should be faster